src/machine/behavior/is/memory/stwu.rs

```diff
@@ -1,6 +1,7 @@
 use machine::state::State;
+use machine::behavior::is::memory::stw;
 
-pub fn stwu(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
+/// store wyde unsigned
+pub fn stwu(state: &mut State, x: u8, y: u8, z: u8) {
+    stw(state, x, y, z);
```

Comment:
The book says that, if an overflow occurs which is not masked, an arithmetic interrupt is triggered. Since we have no time, we don't implement the interrupt but @KarolinePlum and I suggest to catch the overflow and panic if it is not masked (annotated with a FIXME.)